securedrop/source_templates/index.html

```diff
@@ -34,15 +34,16 @@
           </h2>
           <hr class="cut-out" />
           <p>If this is your first time submitting documents to journalists, start here.</p>
-          <a href="/generate" class="btn alt block" id="submit-documents-button"><img id="warning-close" src="{{ url_for('static', filename='i/font-awesome/white/cloud-upload.svg') }}" width="17px" height="17px"> Submit Documents</a>
+          {# adding a break between 'submit' and 'documents' to make both containers equal sized #}
+          <a href="/generate" class="sd-button btn alt block" id="submit-documents-button">SUBMIT<br/>DOCUMENTS</a>
```

Comment:
Looks like these containers are not equal sized in Tor Browser:

![screen shot 2017-01-31 at 1 46 07 pm](https://cloud.githubusercontent.com/assets/7832803/22487270/9883ea8a-e7c1-11e6-8228-d9e1ba226c9f.png)